### DISCUSSION TITLE
Name Idea: tteck's Proxmox VE Helper-Scripts

### DISCUSSION THREAD

**Discussion by Deekerman** [community-scripts/ProxmoxVE#40] [Ideas]
I think that since this has been Founded and Organized originally by Tteck, his name should be included in this. 

Tteck's Proxmox VE Helper-Scripts 

Provided he is ok with it. But its important to know where we started. 

@tteck Thank you for all your help with getting more into the PVE world. You have been invaluable to me and many others. ❤️ 

Deekerman

**Comment by MickLesk** [comment]
tteck said nope in the original repo. But I understand it. I had the same idea at the beginning.

https://github.com/tteck/Proxmox/discussions/4025#discussioncomment-11123543

**Reply by Deekerman** [reply]
What a guy. ok well, it still feels we need to do something.

**Comment by uSlackr** [comment]
Maybe we could noun-ate these and call them “ttechs”!?!

\\Greg
________________________________
